### DISCUSSION TITLE
Introduce SelectableLabel

### DISCUSSION THREAD

**Discussion by enisn** [enisn/UraniumUI#344] [Announcements]
Unfortunately, built-in MAUI labels can't be selected. Sometimes you need a text that can be selected by user. So the following PR implements a new control named `SelectableLabel`. It will be available in `v2.5`

 https://github.com/enisn/UraniumUI/pull/343

It's exactly the same as the regular MAUI label but it's selectable.

![image](https://github.com/enisn/UraniumUI/assets/23705418/414ab8cb-da6f-4abd-acb6-9374ddc9ced7)

**Comment by roughiain** [comment]
Awesome....
